nailgun/entity_mixins.py

```diff
@@ -979,6 +979,9 @@ def update_payload(self, fields=None):
 
         """
         values = self.get_values()
+        updatable_fields = (self.updatable_fields if hasattr(self, 'updatable_fields') else
+                            values.keys())
+        values = {field: value for field, value in values.items() if field in updatable_fields}
```

Comment:
@lhellebr great work!! Thanks for taking the step to implement this for the mixin itself, this looks great, and will be easy to adjust any other entities that are limited in what fields can be updated.

I think this new 'feature' of the mixin should be documented in its docblock, so future users of the class have some instruction to set `entity.updatable_fields` if they want to limit what fields can be modified on the entity.

I would document this in the class docblock, as its relevant for anyone using the mixin class, and they're more likely to see it there first.